src/cryogen_core/compiler.clj

```diff
@@ -493,7 +493,6 @@
     (create-file (path "/" blog-prefix rss-name) (rss/make-channel config posts))
     (println (blue "generating filtered rss"))
     (rss/make-filtered-channels config posts-by-tag)
-    (println (blue "compiling sass"))
```

Comment:
Can you keep this one up here? I like to keep the "top level" messages in this function.